core-compose/src/main/java/com/squareup/workflow/ui/compose/ComposeViewFactory.kt

```diff
@@ -111,23 +112,25 @@ internal class ComposeViewFactory<RenderingT : Any>(
         areEquivalent = StructurallyEqual
     )
 
-    // Models will throw if their properties are accessed when there is no frame open. Currently,
-    // that will be the case if the model is accessed before any other Compose infrastructure has
-    // ran, i.e. if this view factory is the first compose code to run in the app.
-    // I believe that eventually there will be a global frame that will make this unnecessary.
-    FrameManager.ensureStarted()
-
     // Update the state whenever a new rendering is emitted.
     composeContainer.bindShowRendering(
         initialRendering,
         initialViewEnvironment
     ) { rendering, environment ->
       // This lambda will be executed synchronously before bindShowRendering returns.
-      renderState.value = Pair(rendering, environment)
+
+      // Models will throw if their properties are accessed when there is no frame open. Currently,
+      // that will be the case if the model is accessed before any other Compose infrastructure has
+      // ran, i.e. if this view factory is the first compose code to run in the app.
+      // I believe that eventually there will be a global frame that will make this unnecessary.
+      FrameManager.framed {
+        renderState.value = Pair(rendering, environment)
+      }
```

Comment:
Seems more idiomatic to wrap the update itself in a framed, the Compose code does this internally a bunch.